qemu/tests/balloon_minimum.py

```diff
@@ -0,0 +1,41 @@
+import time
+import logging
+
+from virttest import utils_test
+from virttest import error_context
+from virttest.qemu_monitor import QMPEventError
+from qemu.tests.balloon_check import BallooningTestWin
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Balloon negative test, balloon windows guest memory to very small value.
+    1) boot a guest with balloon device.
+    2) enable and check driver verifier in guest.
+    3) evict guest memory to 10M.
+    4) repeat step 3 for many times.
+    5) check guest free memory.
+    """
```

Comment:
Need add doc string for parameters,  other parts look good to me.

Reply:
Added, thanks